upi/metal/main.tf

```diff
@@ -19,69 +19,70 @@ locals {
 }
 
 provider "matchbox" {
-  endpoint    = "${var.matchbox_rpc_endpoint}"
-  client_cert = "${file(var.matchbox_client_cert)}"
-  client_key  = "${file(var.matchbox_client_key)}"
-  ca          = "${file(var.matchbox_trusted_ca_cert)}"
+  version     = "0.3.0"
```

Comment:
we can't use version because there is upstream terraform-provider-matchbox. it is injected into the image https://github.com/openshift/installer/blob/58ecac34ae25797f62fb10504982a8b4562fdaf7/images/installer/Dockerfile.upi.ci#L36-L39

so not setting the version here uses that.

Reply:
should now be fixed by pulling down 0.3.0 in the Dockerfile. I've removed the explicit version requirement here.